### PR TITLE
Add user principals to hello view

### LINKED ISSUE #874: Add user principals to hello view
Along `user` info, show the list of user principals (for groups notably)

### PR DESCRIPTION
**Pull request by gabisurita**
Fixes #874 

- [x] Add documentation.
- [x] Add tests.
- [x] Add a changelog entry.

r? @leplatrem

### CODE REVIEW

File: tests/test_views_hello.py [hunk 1]
```diff
@@ -23,6 +24,17 @@ def test_returns_user_id_if_authenticated(self):
...
 
+    def test_returns_user_principals_if_authenticated(self):
+        group_url = '/buckets/beers/groups/users'
+        group = MINIMALIST_GROUP.copy()
+        group['data']['members'].append(self.principal)
+        self.app.put_json('/buckets/beers', MINIMALIST_BUCKET, headers=self.headers)
+        self.app.put_json(group_url, group, headers=self.headers)
+        response = self.app.get('/', headers=self.headers).json['user']['principals']
+        principals = ('system.Everyone', 'system.Authenticated',
+                      group_url, self.principal.split(':')[1])
```
Comment (Natim) [inline tests/test_views_hello.py]: Why `self.principal.split(':')[1]`?

Comment (Natim) [inline tests/test_views_hello.py]: It should be something like `basicauth:1234` right?

Comment (gabisurita) [inline tests/test_views_hello.py]: ``effective_principals`` actually omits ``basicauth`` prefix. Should the view include ``basicauth:1234`` or just ``1234``.

Comment (leplatrem) [inline tests/test_views_hello.py]: Yes, this is case. 

So, if we want we can put `request.effective_principals + [request.prefixed_userid]` in the hello view instead?

We do some similar black magics here: https://github.com/Kinto/kinto/blob/master/kinto/core/authorization.py#L59

Comment (leplatrem) [inline tests/test_views_hello.py]: I think it's alright since we already have prefixed_userid in `hello[user]`

Comment (Natim) [inline tests/test_views_hello.py]: Note that we also remove the other one: https://github.com/Kinto/kinto/blob/master/kinto/core/authorization.py#L64

**Comment by leplatrem** [comment]
Could you add an entry in the API changelog please?

Thanks :)

**Comment by leplatrem** [comment]
> Please at least remove the principal without the prefix from the list of principals.

Why is it so important? From the Pyramid point of view, it's there!

**Comment by Natim** [comment]
> Why is it so important? From the Pyramid point of view, it's there!
Actually from Kinto point of view it is not and if it is it is an issue because you could have conflict between two Authorizaton policy that would give the same `user_id` principal while it shouldn't be possible

File: docs/api/index.rst [hunk 2]
```diff
@@ -11,12 +11,13 @@ API
 Changelog
 ---------
 
-1.13 (2016-10-20)
+1.13 (2016-10-27)
```
Comment (Natim) [inline docs/api/index.rst]: Isn't this supposed to be `1.14 (unreleased)`

Comment (leplatrem) [inline docs/api/index.rst]: `1.13 unreleased`, but that's acceptable

Comment (Natim) [inline docs/api/index.rst]: It seems to me that 1.13 have been used already: https://github.com/Kinto/kinto/pull/891/files#diff-2c14b00f3393d6e7355fb480528f38e3R22

**Pull request merged**